src/components/common/ColonyActions/ActionDetailsPage/DefaultMotion/MotionPhaseWidget/StakingWidget/StakingInput/StakingSliderMessages/RequiredStakeMessage.tsx

```diff
@@ -24,24 +19,24 @@ const MSG = defineMessages({
   },
 });
 
-const RequiredStakeMessage = () => {
+export interface RequiredStakeMessageProps {
+  totalPercentageStaked: number;
+  remainingToStake: string;
+  userMinStake: string;
+  nativeTokenDecimals: number;
+  nativeTokenSymbol: string;
+}
+
+const RequiredStakeMessage = ({
+  totalPercentageStaked,
+  remainingToStake,
+  userMinStake,
+  nativeTokenDecimals,
+  nativeTokenSymbol,
+}: RequiredStakeMessageProps) => {
   const { watch } = useFormContext();
-  const { colony } = useColonyContext();
-  const { decimals: nativeTokenDecimals, symbol: nativeTokenSymbol } =
-    colony?.nativeToken || {};
-  const {
-    motionStakes: {
-      percentage: { yay: yayPercentageStaked, nay: nayPercentageStaked },
-    },
-    remainingStakes: [nayRemaining, yayRemaining],
-    userMinStake,
-  } = useStakingWidgetContext();
-  const isObjection = false;
-  const remainingToStake = isObjection ? nayRemaining : yayRemaining;
   const sliderAmount = watch(SLIDER_AMOUNT_KEY);
-  const isUnderThreshold =
-    sliderAmount <
-    10 - Number(isObjection ? nayPercentageStaked : yayPercentageStaked);
+  const isUnderThreshold = sliderAmount < 10 - totalPercentageStaked;
```

Comment:
Threshold should be a constant within this file _(or even better, the constants file)_